sr_description/hand/xacro/finger/distal/distal.urdf.xacro

```diff
@@ -34,6 +34,7 @@
 
   <xacro:include filename="$(find sr_description)/hand/xacro/finger/distal/distal.gazebo.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/finger/distal/distal.transmission.xacro" />
+  <xacro:include filename="$(find sr_mst)/xacro/f_distalXL.urdf.xacro" />
```

Comment:
This is introducing a dependency on sr_mst. This package cannot depend on a package that might not be present in the workspace.
We should probably move the relevant files to sr-description.

Reply:
Updated now